TypeCobol.LanguageServer/TypeCobolCustomLanguageServer/TypeCobolCustomLanguageServer.cs

```diff
@@ -23,29 +24,42 @@ public TypeCobolCustomLanguageServer(IRPCServer rpcServer, Queue<MessageActionWr
         /// </summary>
         public bool UseOutlineRefresh { get; set; }
 
-        protected override void OnShutdown()
-        {
-            if (UseOutlineRefresh && this.Workspace.DocumentModifiedEvent != null)
-                this.Workspace.DocumentModifiedEvent -= DocumentModified;
+        /// <summary>
+        /// Are we using the CFG view in the client.    
+        /// </summary>
+        public bool UseCfgDfaDataRefresh { get; set; }
+        /// <summary>
+        /// True if The server is executed in LSR TestMode
+        /// </summary>
+        public bool InRobotLsrTestMode { get; set; }
```

Comment:
Le flag peut désormais être supprimé, on utilise que `InLsrClientMode`.

Reply:
Now both can be used.

Reply:
Originally, we only had `TypeCobolServerHost.LsrMode` which indicate whether we are using the robot or not and it was limited to `TypeCobolServerHost` class. Now we have to know that in `TypeCobolCustomLanguageServer` to select the correct output type for CFG data. So why not just add a new property `LsrMode` in `TypeCobolCustomLanguageServer` and initialize it with the value from `TypeCobolServerHost.LsrMode` ?

In my opinion, we don't need a new option `lsrclient`, we just need to let the server know that the robot is used and we already have the option for that.

Reply:
My point of view is that it can be  interresting to know if the server is called from for instance CobolEditEI or it is called from the Test execution mode. LsrMode was and is the InRobotLsrTestMode option, and was only set  by test execution, not from the client point.

Reply:
So if the client uses the `-r` option, we would know that we are in robot test mode but we could not distinguish between unit test and real client. Is that a problem ? I mean the code should never know that it is running in unit test context.

Reply:
I don't understand either why we need 2 options.
`InRobotLsrTestMode` and `InLsrClientMode` are exactly in the same way in TypeCobolCustomLanguageServer#UpdateCfgDfaInformation.